SpatialDepthWiseConvolution.lua

```diff
@@ -118,7 +127,7 @@ end
 
 function SpatialDepthWiseConvolution:__tostring__()
    local s = string.format('%s(%d -> %d, %dx%d', torch.type(self),
-         self.nInputPlane, self.nOutputPlane, self.kW, self.kH)
+         self.nInputPlane, self.depthwiseMultiplier, self.kW, self.kH)
```

Comment:
add a :read method to this file similar to:
https://github.com/torch/nn/blob/master/BatchNormalization.lua#L184-L192

Add a `__version = 2` similar to https://github.com/torch/nn/blob/master/BatchNormalization.lua#L33

Then, in the `:read`, add:
```
   parent.read(self, file)
   if version < 2 then
       self.depthwiseMultiplier = self.nOutputPlane
       self.nOutputPlane = self.nInputPlane * self.depthwiseMultiplier;
   end
```